packages/distributor/solana/utils.ts

```diff
@@ -5,8 +5,12 @@ import { Connection, Keypair, PublicKey, Transaction, VersionedTransaction } fro
 import { ContractError, divCeilN } from "@streamflow/common";
 import { ConfirmationParams, signAndExecuteTransaction, ThrottleParams } from "@streamflow/common/solana";
 
-import { fromTxError } from "./generated/errors";
-import { ONE_IN_BASIS_POINTS } from "./constants";
+import { fromTxError } from "./generated/errors/index.js";
+import { ONE_IN_BASIS_POINTS, ALIGNED_DISTRIBUTOR_PREFIX } from "./constants.js";
+
+export const getAlignedDistributorPDA = (programId: PublicKey, distributor: PublicKey): PublicKey => {
```

Comment:
since it is public, let's align `PDA` case to others 
```suggestion
export const getAlignedDistributorPda = (programId: PublicKey, distributor: PublicKey): PublicKey => {
```